setup.py

```diff
@@ -8,11 +8,17 @@ def _cut_version_number_from_requirement(req):
     return req.split()[0]
 
 
-def read_version():
-    with open("./cgp/__version__.py") as f:
-        line = f.read()
-        match = re.findall(r"[0-9]+\.[0-9]+\.[0-9]+", line)
-        return match[0]
+def read_meta(meta):
+    """
+    Find __"meta"__ from init file.
```

Comment:
```suggestion
    Find __"meta"__ in init file.
```
or "Read __"meta"__ from init file", don't have a strong preference